tests/dynamic_sharing_service/dynamic.scenario.builder.spec.js

```diff
@@ -1,12 +1,6 @@
-const {
-  expect,
-} = require('chai');
+const { expect } = require('chai');
 
-const {
-  DynamicScenarioBuilder,
-  DynamicPolicyBuilder,
-  ExtensionBuilder,
-} = require('../../');
+const { DynamicScenarioBuilder, DynamicPolicyBuilder, ExtensionBuilder } = require('../../');
```

Comment:
Same with these - if we are to change the formatting of files through an auto-formatter, I would rather we did them in one go, so as not to detract away from other PRs.

Reply:
[Fixed here.](https://github.com/getyoti/yoti-node-sdk/commit/067bdfe0f960725d452e43c112f4adbf47f3c8c1) . I have disabled prettier until further notice